evap/evaluation/templates/infotext.html

```diff
@@ -0,0 +1,13 @@
+{% if infotext.title and infotext.content %}
```

Comment:
Personally, I'd prefer an or here: If one is empty, the other should still be shown.

On the staff edit page, there should probably be a check then that both are filled or both are empty.

Reply:
Moreover I'd say, if the content is empty, the box should be closed at all times